resources/services/frontdoor/front_doors.go

```diff
@@ -0,0 +1,910 @@
+									},
+									{
+										Name:        "transforms",
+										Description: "List of transforms",
+										Type:        schema.TypeStringArray,
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			{
+				Name:        "azure_front_door_routing_rules",
+				Description: "Routing rules represent specifications for traffic to treat and where to send it, along with health probe information.",
+				Resolver:    fetchFrontdoorFrontDoorRoutingRules,
+				Columns: []schema.Column{
+					{
+						Name:        "front_door_cq_id",
+						Description: "Unique CloudQuery ID of azure_front_doors table (FK)",
+						Type:        schema.TypeUUID,
+						Resolver:    schema.ParentIdResolver,
+					},
+					{
+						Name:        "resource_state",
+						Description: "Resource state",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("RoutingRuleProperties.ResourceState"),
+					},
+					{
+						Name:        "accepted_protocols",
+						Description: "Protocol schemes to match for the rule",
+						Type:        schema.TypeStringArray,
+						Resolver:    schema.PathResolver("RoutingRuleProperties.AcceptedProtocols"),
+					},
+					{
+						Name:        "patterns_to_match",
+						Description: "The route patterns of the rule",
+						Type:        schema.TypeStringArray,
+						Resolver:    schema.PathResolver("RoutingRuleProperties.PatternsToMatch"),
+					},
+					{
+						Name:        "enabled_state",
+						Description: "Whether the rule is enabled",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("RoutingRuleProperties.EnabledState"),
+					},
+					{
+						Name:        "route_configuration",
+						Description: "A reference to the routing configuration",
+						Type:        schema.TypeJSON,
+						Resolver:    resolveFrontDoorRoutingRulesRouteConfiguration,
+					},
+					{
+						Name:          "rules_engine_id",
+						Description:   "ID of a specific Rules Engine Configuration to apply to the route",
+						Type:          schema.TypeString,
+						Resolver:      schema.PathResolver("RoutingRuleProperties.RulesEngine.ID"),
+						IgnoreInTests: true,
+					},
+					{
+						Name:          "web_application_firewall_policy_link_id",
+						Description:   "ID of the Web Application Firewall policy for each routing rule (if applicable)",
+						Type:          schema.TypeString,
+						Resolver:      schema.PathResolver("RoutingRuleProperties.WebApplicationFirewallPolicyLink.ID"),
+						IgnoreInTests: true,
+					},
+					{
+						Name:        "name",
+						Description: "Resource name",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "type",
+						Description: "Resource type",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "id",
+						Description: "Resource ID",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("ID"),
+					},
+				},
+				Relations: []*schema.Table{
```

Comment:
maybe an array of strings? instead of a relation?

Reply:
done